lib/voice/VoiceConnection.js

```diff
@@ -262,7 +262,7 @@ class VoiceConnection extends EventEmitter {
                             this.disconnect(err);
                         }
                     });
-                    const udpMessage = Buffer.allocUnsafe(70);
+                    const udpMessage = Buffer.allocUnsafe(74);
```

Comment:
In my testing, I also had to change byte range being read on line 244 as well.

Reply:
That padded a bunch of null characters at the end of the local IP. It still worked, but probably want a way to remove those.

Reply:
Was pretty sure that I needed this change, but looks like they reverted back to the 70 byte payload in the meantime, so I can't really confirm. Ignore this comment I guess

Reply:
Just tested this, the response structure is still the same as in the docs.